worlds/seaofthieves/Regions/Regions.py

```diff
@@ -0,0 +1,88 @@
+import copy
+import typing
+
+from BaseClasses import MultiWorld, Region, Entrance
+from worlds.seaofthieves.Locations.Locations import *
```

Comment:
```suggestion
from ..Locations.Locations import *
```
Use relative imports when importing from within your own world.